src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -95,9 +138,23 @@
     - name: Get an access token to use with MS Graph API
       text: >
         az account get-access-token --resource-type ms-graph
+    - name: >
+        Get an access token for a particular resource.  For a list of Virtual Network available service endpoints,
```

Comment:
Single space should suffice.
```suggestion
        Get an access token for a particular resource. For a list of Virtual Network available service endpoints,
```